central/graphql/resolvers/node_components.go

```diff
@@ -0,0 +1,108 @@
+package resolvers
+
+import (
+	"context"
+	"time"
+
+	"github.com/graph-gophers/graphql-go"
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/central/metrics"
+	"github.com/stackrox/rox/pkg/features"
+	pkgMetrics "github.com/stackrox/rox/pkg/metrics"
+	"github.com/stackrox/rox/pkg/search"
+	"github.com/stackrox/rox/pkg/utils"
+)
+
+func init() {
+	schema := getBuilder()
+	utils.Must(
+		// NOTE: This list is and should remain alphabetically ordered
+		schema.AddType("NodeComponent", []string{
+			"fixedIn: String!", // is this needed?
+			"id: ID!",
+			"nodeComponentLastScanned: Time",
+			"license: License",
+			"location(query: String): String!",
+			"name: String!",
+			"nodes(query: String, scopeQuery: String, pagination: Pagination): [Node!]!",
+			"nodeCount(query: String, scopeQuery: String): Int!",
+			"plottedNodeVulnerabilities(query: String): PlottedVulnerabilities!",
+			"priority: Int!",
+			"riskScore: Float!",
+			"source: String!", // is this infrastructure ?
+			"topNodeVulnerability: NodeVulnerability",
+			"unusedVarSink(query: String): Int",
+			"version: String!",
+			"nodeVulnerabilities(query: String, scopeQuery: String, pagination: Pagination): [NodeVulnerability]!",
+			"nodeVulnerabilityCount(query: String, scopeQuery: String): Int!",
+			"nodeVulnerabilityCounter(query: String): VulnerabilityCounter!",
```

Comment:
These aren't in the right spot, should be with the `n`s

Reply:
Good catch